src/main/java/net/irisshaders/lilybot/commands/support/UpdateDrivers.java

```diff
@@ -11,41 +10,96 @@
 import java.time.Instant;
 
 public class UpdateDrivers extends SlashCommand {
-    public UpdateDrivers() {
-        this.name = "update-drivers";
 
+    public UpdateDrivers() {
+        this.name = "drivers";
```

Comment:
Should be update-drivers still

Reply:
Can't seem to have the dash because wierd JDA but other than that it is now updatedrivers